handler/CMakeLists.txt

```diff
@@ -137,9 +137,12 @@ if(CRASHPAD_WER_ENABLED)
     target_link_libraries(crashpad_wer
         PRIVATE
             $<BUILD_INTERFACE:crashpad_interface>
+        PUBLIC
+            crashpad_compat
```

Comment:
If we add this as `PUBLIC` here, we need to also make it part of an export set. Wouldn't it be enough to add `crashpad_compat` the same way as the `crashpad_interface` above?

Reply:
> Besides a few fixes, this LGTM. Can you also provide the required CMake CLI parameters you used to get this to run?

I have opened a PR in the parent sentry-native repo to add CI for llvm-mingw builds (including testing this crashpad integration): https://github.com/getsentry/sentry-native/pull/797